backend/respositories/user.repository.js

```diff
@@ -60,30 +36,18 @@ const getUserByEmail = async (email) => {
         return null;
     }
 
-    user = {
-        userid: user.userid,
-        name: user.name,
-        email: user.email,
-        username: user.username,
-        password: user.password,
-        createdAt: user.createdAt,
-        updatedAt: user.updatedAt
-    };
-
-    return new UserDTO(user);
+    return user;
 }
 
 const updateUser = async (user, updateFields) => {
 
-    var result = await User.update(updateFields, {
-        where: { userid: user.userid }
-    });
+    var updatedUser = await user.update(updateFields);
```

Comment:
always declare variables with const, if you realize that the value of the variable needs to change, go back and change it to let.

Reply:
changed